ui/src/app/utils/strings.ts

```diff
@@ -0,0 +1,3 @@
+export const extractStringByKey = (key?:string) => {
```

Comment:
don't use hard coded value, get the split key (.) and index in arg 
extractStringByKey => extractStringByKeyPos (str, key, pos)
@manikanta-reddy-t